src/step_03.py

```diff
@@ -30,23 +30,25 @@ def count_values(df):
                                                  input=month_values_cfg,
                                                  output=nb_of_values_cfg)
 
-pipeline_cfg = Config.configure_pipeline(id="my_pipeline",
-                                         task_configs=[task_filter_cfg,
-                                                       task_count_values_cfg])
 
-scenario_cfg = Config.configure_scenario(id="my_scenario",
-                                         pipeline_configs=[pipeline_cfg])
+scenario_cfg = Config.configure_scenario_from_tasks("my_scenario",
+                                                    [task_filter_cfg,
+                                                     task_count_values_cfg])
```

Comment:
I like to see keyword parameters being used when I'm still learning a package so that I know what the function is expecting. Please correct my hanging indent if it is wrong -- I'm not familiar with formatting code this way.

```suggestion
scenario_cfg = Config.configure_scenario_from_tasks(id="my_scenario",
                                                    task_configs=[task_filter_cfg,
                                                                  task_count_values_cfg])
```